Cargo.lock

```diff
@@ -832,9 +832,9 @@ dependencies = [
 
 [[package]]
 name = "kvm-ioctls"
-version = "0.19.0"
```

Comment:
We need to update this in the `src/vmm/Cargo.toml` as well.

Reply:
Not really, since Cargo.lock is the source of truth, and when regenerating it, it will pick the newest versions compatible with the tomls. But can update anyway.